Bottom: d7cca3d171346daf6ebc571f708918943b88cad4
Top:    eef93bd95fa7c70fbc57533aac0a29ce229a49ed
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 17:31:01 +0000

Refresh of refactor-polygon-copy

---

diff --git a/src/polygon1.c b/src/polygon1.c
index ca7a7e7..55772a5 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2965,7 +2965,7 @@ poly_CopyContour (PLINE ** dst, PLINE * src)
 /**********************************************************************/
 /* polygon routines */
 
-static bool
+static BOOLp
 poly_Copy1 (POLYAREA * dst, const POLYAREA * src)
 {
   PLINE *cur, **last = &dst->contours;
@@ -2976,12 +2976,12 @@ poly_Copy1 (POLYAREA * dst, const POLYAREA * src)
   for (cur = src->contours; cur != NULL; cur = cur->next)
     {
       if (!poly_CopyContour (last, cur))
-        return false;
+        return FALSE;
       r_insert_entry (dst->contour_tree, (BoxType *) * last, 0);
       last = &(*last)->next;
     }
 
-  return true;
+  return TRUE;
 }
 
 BOOLp
